### PR TITLE
Feature bar volume

### PR DESCRIPTION
**Pull request by aahill50**
Boom!

@lilianammmatos Most of this is your stuff, I just expanded it to add support for any additional speakers we may want to add in the future. The max vols and adjustment ratios are separate for each and easy to edit.

### CODE REVIEW

**Comment by lilianammmatos** [comment]
did you test this?

**Comment by aahill50** [comment]
Tested locally, I don't know how to do that magic ssh business

File: src/scripts/sonos.js [hunk 1]
```diff
@@ -31,33 +31,57 @@ var DEFAULT_MAX_VOL = 50;
 var MAX_PUBLIC_VOLUME = 100;
 var MIN_PUBLIC_VOLUME = 0;
 
-var maxVol = DEFAULT_MAX_VOL;
+var MAX_VOLS = {
+    main: DEFAULT_MAX_VOL,
+    bar: 60
+};
+
+var VOL_RATIOS = {
```
Comment (hellatan) [inline src/scripts/sonos.js:39]: `var`?! ew

Comment (aahill50) [inline src/scripts/sonos.js:39]: 😒

**Comment by lilianammmatos** [comment]
Can we get `dibsy bar volume +/-` working before merging?

File: src/scripts/sonos.js [hunk 2]
```diff
@@ -31,33 +31,57 @@ var DEFAULT_MAX_VOL = 50;
...
 
-var sonosInstance;
+var sonosInstance = [];
 
 var volumeIsLocked = false;
 
-function getSonos() {
+var SONOS_HOST_MAP = {
+    main: process.env.HUBOT_SONOS_HOST,
+    bar : '192.168.96.159'
```
Comment (lilianammmatos) [inline src/scripts/sonos.js]: Can we get this host in the env vars? cc @cmckendry

Comment (lilianammmatos) [inline src/scripts/sonos.js]: `process.env.HUBOT_SONOS_BAR_HOST` ?

**Comment by aahill50** [comment]
+/- is fixed now

File: src/scripts/sonos.js (hunk 2 above)
Comment (cmckendry) [inline src/scripts/sonos.js]: HUBOT_SONOS_BAR_HOST: "192.168.96.159",

Comment (lilianammmatos) [inline src/scripts/sonos.js]: @aahill50 can you update to use env var?

Comment (aahill50) [inline src/scripts/sonos.js]: Done

**Comment by lilianammmatos** [comment]
```
liliana at Liliana-2 in ~/projects/hubot-scripts on master
$ npm publish
+ hubot-1stdibs@7.0.0
```

**Pull request merged**